simd/simd-memtest/compiler.h

```diff
@@ -52,6 +60,9 @@ typedef int64_t __int64;
 #else
 #define OMP_PARALLEL_FOR
 static inline int omp_get_thread_num() { return 0; }
```

Comment:
Move all the stub functions here, one was redefined in another file `driver.c` causing an error.